src/DIPS.Xamarin.UI/Controls/Tag/Tag.cs

```diff
@@ -0,0 +1,202 @@
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Controls.Tag
+{
+    /// <summary>
+    ///     A <c>Frame</c> wrapping a <c>Label</c> to allow setting border radius for the <c>Label</c>.
+    /// </summary>
+    public class Tag : Frame
```

Comment:
Inherit fra contentview to keep properties protected. If consumers need to set something we should add our own property that changes the internal properties of the frame. That's how our other Controls are developed.

Reply:
I did it to the Label wrapped in, it wasn't so appealing to do the Frame, coz have to expose the Frame properties... i'll wrap the frame, to make it consistent